mirage-bootvar-solo5.opam

```diff
@@ -14,12 +14,12 @@ tags: [
   "org:mirage"
 ]
 build: [
-  ["ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%"]
+  ["dune" "subst"] {pinned}
```

Comment:
What does this do?

Reply:
This changes watermarks such as `VERSION`, `PKG_MAINTAINER`, ... in the code with actual values, when the package is pinned.
You can see [dune subst documentation](https://dune.readthedocs.io/en/latest/usage.html#dune-subst) for further information.
There are no such watermarks in this repo, but as for `dune runtest`, I added it anyway in case some watermarks are added in the future.